onnxruntime/core/providers/cpu/tensor/identity_op.h

```diff
@@ -43,7 +43,18 @@ class IdentityOp final : public OpKernel {
     }
 
     if (is_dropout) {
-      context->Output(1, std::vector<int64_t>());
+      Tensor* mask = context->Output(1, shape);
+      // a 'nullptr' returned would make it an unused optional output
+      if (mask != nullptr) {
+        // Opset 7 differs with Opset 10 in that the type of the 'mask'
+        // output is tied with the type of the input in Opset 7 whereas
+        // the type of 'mask' in Opset 10 is 'bool' always
+        // so we have a common solution
+        void* mask_data = mask->MutableDataRaw();
+        // In 'test'/'inference' mode, there are no input values dropped out
+        // so fill the buffer with 0/false
+        memset(mask_data, 0, mask->Size());
```

Comment:
SizeInBytes()?

Reply:
When I cherry-picked initially, the build broke, I think because the API was originally Size() which got renamed to SizeInBytes() in master. Since this branch doesn't have that change, I kept it Size(). Do you want to pick the API change as well ?

Reply:
Ok, I just wanted to double check that it wasn't the number of elements. (yeah, having a Size method here is really confusing, vs a Count or SizeInBytes which is much clearer)